lib/senders/oauth_client_info.js

```diff
@@ -4,16 +4,10 @@
 
 'use strict'
 
-const request = require('request')
 const Keyv = require('keyv')
-const Joi = require('joi')
 
-const P = require('../promise')
-const { DISPLAY_SAFE_UNICODE } = require('../routes/validators')
+module.exports = (log, config, oauthdb) => {
```

Comment:
We already had some code that was making backend calls to oauth-server, albeit unauthenticated ones.  So I took the opportunity to try to consolidate it behind an abstraction layer of an "oauthdb" lib (which is really just a proxy layer for now, but might become an actual DB layer as our smooshing plans progress).